entity/src/relationship.rs

```diff
@@ -45,4 +46,12 @@ pub enum Relationship {
     DevToolOf,
     #[sea_orm(num_value = 13)]
     DescribedBy,
+    #[sea_orm(num_value = 14)]
+    PackageOf,
+}
+
+impl fmt::Display for Relationship {
```

Comment:
I guess this works :) There's also a crate called `strum`, which gives you a set of `derive`s that can do the same for you.